### ISSUE TITLE
are any layers in ResNet50 can't be set layer.trainable = True?

### ISSUE THREAD

**Issue by Jubird915** [keras-team/keras#11141]
when I retrain the ResNet50 network model for imagenet, I want to train all the weights.
so I set all the layers.trainable= True, but when it started training, suddenly killed the training process, and has no error reported.

    model = keras.applications.resnet50.ResNet50()
    print("len layers={}".format(len(model.layers)))
    model.layers.pop()
    for layer in model.layers:
        layer.trainable=True

when run this training code, the log as below:
Instructions for updating:

Future major versions of TensorFlow will allow gradients to flow
into the labels input on backprop by default.

See @{tf.nn.softmax_cross_entropy_with_logits_v2}.

Epoch 1/100
已杀死

but when I changed code  layer.trainable=False, it ran successfully.
confused that is there any layers in ResNet50 can't be set layer.trainable=True?

**Comment by gabrieldemarmiesse** [comment]
Currently keras is tested against TF 1.9. Please use this version. We will address those issues as we bump the version of tensorflow we test against. No worries.

By default, all the weights are trainable. No need set the trainable flag manually to true. If you do, don't forget to recompile again.
